grafana-plugin/src/state/plugin/index.ts

```diff
@@ -213,25 +211,19 @@ class PluginState {
     }
   };
 
-  static installPlugin = async <RT = CloudProvisioningConfigResponse>(
-    selfHosted = false
-  ): Promise<InstallPluginResponse<RT>> => {
-    const { key: grafanaToken } = await this.createGrafanaToken();
-    await this.updateGrafanaPluginSettings({ secureJsonData: { grafanaToken } });
```

Comment:
just consolidate this call to `this.updateGrafanaPluginSettings` (to save the `grafanaToken` in `secureJsonData`) to the `createGrafanaToken` method above